src/value.rs

```diff
@@ -35,11 +43,10 @@ pub enum Value {
   F64(f64),
```

Comment:
I've refrained from asking for far too long but. why not `float(f64)` and `int(isize)`?